stripe/api_resources/application.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# File generated from our OpenAPI spec
+from __future__ import absolute_import, division, print_function
+
+from stripe.stripe_object import StripeObject
+from typing import Optional
+from typing_extensions import Literal
+
+
+class Application(StripeObject):
```

Comment:
This feels like a separate decision from "annotating resource properties". Are we saying that we'll start generating all top level resources in type-annotated libraries?

Reply:
What's the alternative? How do we give a type to a field like

https://github.com/stripe/stripe-python/pull/1026/files/08adc64efa3d4721987cb919724375c070b3cd59#diff-e8276cf621b514c555d80becdf96a85735068568b3b7bdcce05964dbd66234e9R66

?

Here's the analogous definition in `stripe-node`. https://github.com/stripe/stripe-node/blob/master/types/Applications.d.ts

If you like, I can postpone the decision for how to handle resource references to another PR and do `any` in these places for now.

Reply:
>What's the alternative? How do we give a type to a field like

`StripeObject`? similar to how we type other fields that don't have generated types in Python. And what we already do in PHP today.

I don't want to postpone it. I want to explicitly discuss and make it and file followups (like doing the same thing in PHP).